db/schema_applier.cc

```diff
@@ -639,17 +639,14 @@ static future<> merge_tables_and_views(distributed<service::storage_proxy>& prox
         co_await max_concurrent_for_each(tables_diff.created, max_concurrent, [&] (global_schema_ptr& gs) -> future<> {
             co_await db.add_column_family_and_make_directory(gs, replica::database::is_new_cf::yes);
         });
-        co_await max_concurrent_for_each(views_diff.created, max_concurrent, [&] (global_schema_ptr& gs) -> future<> {
-            co_await db.add_column_family_and_make_directory(gs, replica::database::is_new_cf::yes);
-        });
-    });
-    co_await db.invoke_on_all([&](replica::database& db) -> future<> {
         std::vector<bool> columns_changed;
```

Comment:
> schema_applier: make updates to tables and views atomic 

If this is vital for correctness of materialized views, please leave a comment somewhere here which explains that it is important not to break the atomicity and why.

Reply:
Ok, I added a comment